businessRules/users.js

```diff
@@ -0,0 +1,56 @@
+const { v4 } = require('uuid');
+const md5 = require('md5');
+const { createUser } = require('../models/users');
+const { getUsers, setUsers } = require('../utils/storage');
+
+function registerUser(userName, password, firstName, lastName) {
+    const noFirstName = 'Anonymous';
+    const noLastName = 'Incognito';
+
+    if (!firstName) {
+        firstName = noFirstName;
+    }
+
+    if (!lastName) {
+        lastName = noLastName;
+    }
+
+    createUser(userName, password, firstName, lastName);
+}
+
+function login(req, res) {
+    let users = getUsers();
+
+    const userName = req.body.userName;
+    const password = md5(req.body.password);
+
+    const user = users.find(
+        (u) => u.userName === userName && u.password === password
```

Comment:
Do not use single character names - this will complicate refactoring in the future

Reply:
Fixed. Tested